src/plugins/intel_gpu/src/runtime/execution_config.cpp

```diff
@@ -60,7 +61,8 @@ bool requires_new_shape_infer(const std::shared_ptr<ov::Node>& op) {
             return true;
     }
 
-    if (ov::is_type<ov::op::v5::GRUSequence>(op) || ov::is_type<ov::op::v5::LSTMSequence>(op) || ov::is_type<ov::op::v4::LSTMCell>(op)) {
```

Comment:
Could you please add functional test?

Reply:
I think it would be enough to check 3D QKV case with attention mask for static model